src/api/MoneyMoovApiClient.ts

```diff
@@ -0,0 +1,73 @@
+import { deletePaymentRequest, getPaymentRequests, voidPaymentRequest } from "./PaymentRequestClient";
+import { PaymentRequestPageResponse } from "./types/ApiResponses";
+import { ApiError } from "./types/ApiResponses";
+
+class MoneyMoovApiClient {
+  apiUrl: string;
+  token: string;
+
+  /**
+   * @param apiUrl The api base url
+   * @param token The token
+   */
+  constructor(apiUrl: string, token: string) {
+    this.apiUrl = apiUrl;
+    this.token = token;
+  }
+
+  /**
+   * Gets a paged list of Payment Requests
+   * @param apiUrl The base api url. E.g https://api.nofrixion.com/api/v1
+   * @param token The auth token.
+   * @param page The first page. Default is 1
+   * @param size The page size. Default is 20
+   * @returns A PaymentRequestPageResponse if successful. An ApiError if not successful.
+   */
+  getPaymentRequests = async (
+    apiUrl: string,
+    token: string,
+    page: number = 1,
+    size: number = 20
+  ): Promise<{
+    data?: PaymentRequestPageResponse | undefined;
+    error?: ApiError | undefined;
+  }> => {
+    return await getPaymentRequests(apiUrl, token, page, size);
+  };
+
+  /**
+   * Deletes a Payment Request
+   * @param baseApiUrl The base api url. E.g https://api.nofrixion.com/api/v1
+   * @param token The auth token.
+   * @param paymentRequestId The Payment Request Id
+   * @returns
+   */
+  deletePaymentRequest = async (
+    baseApiUrl: string,
+    token: string,
+    paymentRequestId: string
+  ): Promise<{
+    error?: ApiError | undefined;
+  }> => {
+    return await deletePaymentRequest(baseApiUrl, token, paymentRequestId);
+  };
+
+  /**
+   * Voids a Payment Request

```

Comment:
Might exceed PR's porpoise, but I believe that'd be great if we could add better description for the methods.

For example, in void we'd clarify what "voiding" a payment request means.

Something like

> This function voids a transaction made through a payment request, canceling or invalidating it after it has been initiated but before it has been processed.
> The voided transaction is effectively erased from the record as if it never happened, and any associated funds are returned to their original source.
> This function is useful for correcting mistakes, such as entering the wrong amount or sending a payment to the wrong recipient.
> Note that voiding a transaction is different from issuing a refund, which occurs after the transaction has been processed and involves returning funds to the customer's account. 


I know this might be too much, but I curious to hear your thoughts on this.